frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -1084,7 +1084,11 @@ frappe.views.QueryReport = class QueryReport extends frappe.views.BaseList {
 		], ({ file_format, include_indentation }) => {
 			this.make_access_log('Export', file_format);
 			if (file_format === 'CSV') {
-				const column_row = this.columns.map(col => col.label);
+				const column_row = this.columns.map(col => {
```

Comment:
Use `.filter` and then `.map`. Using just map will return undefined for hidden columns. Is it intentional?